isofit/core/common.py

```diff
@@ -743,6 +743,31 @@ def ray_start(num_cores, num_cpus=2, memory_b=-1):
         result = subprocess.run(base_args, capture_output=True)
 
 
+def match_statevector(
```

Comment:
This function is used throughout the fileio and analytical line routines to match the pixel statevector agains the image statevector.  The matching is done against the string values of the state element name, not the index.